sitemap.py

```diff
@@ -173,3 +181,22 @@ def sitemap():
 
 
 sitemap()
+
+
+# Takes in a string containing the hidden files and folders and splits them based on commas.
+# Writes them to the corresponding production and non-production .txt files from config
+def hidden_files():
+    hidden = hidden_pages.split(",")
+    with open(config.HIDDEN_FILE, 'w') as file:
+        file.write("\n".join(hidden))
+    with open(config.HIDDEN_PRODUCTION_FILE, 'w') as file:
+        file.write("\n".join(hidden))
+
+    hidden = hidden_folders.split(",")
+    with open(config.HIDDEN_FOLDER, 'w') as file:
+        file.write("\n".join(hidden))
+    with open(config.HIDDEN_PRODUCTION_FOLDER, 'w') as file:
```

Comment:
Might be worth to change the config text file location variables to be `HIDDEN_PAGES_FILE` and `HIDDEN_FOLDERS_FILE` potentially. Might be a bit more descriptive.

Reply:
Good idea! I just added that